### ISSUE TITLE
Normalized Evidence Raster BL

### ISSUE THREAD

**Issue by shelbysawyer** [Riverscapes/RiverscapesXML#90]
Add the following symbology keys to the BL:

- [x] norm_slope
- [x] norm_hand
- [x] norm_channel
- [x] norm_flow

@joewheaton For the normalized rasters (slope, HAND, channel distance, flow area distance), I have the .lyr files (norm_slope.lyr, norm_hand.lyr, norm_channel.lyr, norm_flow.lyr) ready to go, but I'm not sure how to go about building their symbology into the BL. I've plucked the chunk of code out from the BL where it would live, along with the lines of code that specify the evidence rasters. As the normalized evidence raster is a 'repeater' (guessing this just means its duplicated in the TOC?), should I just imitate this same code structure for the normalized evidence raster symbology? A little guidance here would be helpful.

```
          <Node label="Evidence Rasters">
            <Children collapsed='true'>
              <Node xpathlabel="Name" xpath="Inputs/Raster[@id='SLOPE_RASTER']" type="raster" symbology="slope" transparency="40" />
              <Node label="HAND Raster" xpath="Intermediates/Raster[@id='HAND_RASTER']" type="raster" symbology="hand" transparency="40" />
              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='CHANNEL_DISTANCE']" type="raster" symbology="channeldistance" />
              <Node xpathlabel="Name" xpath="Intermediates/Raster[@id='FLOW_AREA_DISTANCE']" type="raster" symbology="flowarea" />
            </Children>
          </Node>
          <Repeater label="Normalized Evidence Rasters" xpath="Intermediates/Raster[contains(@id, 'NORMALIZED_')]">
            <Node xpathlabel="Name" type="raster" symbology="" transparency="40" />
          </Repeater>
```

**Comment by joewheaton** [comment]
See #83 comments.

**Comment by joewheaton** [comment]
Have you seen [this](http://rave.riverscapes.xyz/business-logic.html#node)? 

It is simply a matter of using the exact same case and spelling for the `symbology` tag as you use for the `*.lyr` and `*.qml` and putting those in the correct location (i.e. under VBET) in RiverscapesXML repo.

The repeater is for something else... maybe @philipbaileynar can explain why that is showing up in there.

**Comment by shelbysawyer** [comment]
I updated the business logic to specify the symbology for the normalized rasters. @tyler1218hatch Helped me go through and make sure everything looked good-- making sure the paths were correct, that the symbology tags matched the `.lyr` files and making sure that they are in the correct place. Still, they don't display properly. We were unable to get to the bottom of what is causing this issue. I'm going to try downloading the new project again and see if that happens to solve whatever is going on.

We played around and tested to make sure the project is pulling from the XML properly (with the updates), and it is.. apart from the '.lyr' symbology. Dragging the `.lyr` files into ArcMap showed that they all display properly. Not sure what the hangup is here.

**Comment by joewheaton** [comment]
Thanks @shelbysawyer. Please record a video and post a YouTube link to it here of exactly what is "not working". Please show us the paths, what it should like, and what "it is doing" now. The above information is a good start, but a little hard to follow.  Please also provide the datasets (e.g. RS Project), layer files and business logic you are using. If I can't figure it out from that we'll ask @philipbaileynar for his help.

**Comment by shelbysawyer** [comment]
Sure @joewheaton-- [here](https://youtu.be/0Zk9R5nUlSA) is a video covering the issue with `.lyr` files not being properly pulled into projects. It covers the behavior that I am seeing with normalized raster symbology and what the layers should look like, as well as the paths of the relevant files. Let me know if I need to provide more information.

[Here](https://data.riverscapes.xyz/#/PUBLIC_DATA/a40a1e2f-54f4-4da2-ae3c-feab6ff5973c) is the project I am using.

**Comment by joewheaton** [comment]
Sorry I missed this @shelbysawyer. Your video was really helpful. I could see from https://youtu.be/0Zk9R5nUlSA?t=85 that you had your layer files in your `c:\0_GIT\RivercapesXML\Symbology\ESRI\` , which is just where you have clone the RiverscapesXML repo. Your problem is that "ArcRAVE is not finding them" because they are NOT in the "correct place" for it to find them. They are in the correct place when you want to commit those changes to the remote GitHub repository. However, to test them, they need to be in [one of the places RAVE looks for them](http://rave.riverscapes.xyz/business-logic.html#where-does-rave-look-for-business-logic-xml-files):

1. User-loaded from Customize Project Hierarchy command (Priority 1)
2. In the root directory of the current project (Priority 2)
3. Local user default over-ride for that project type (Priority 3)
4. Default what ships with that version of RAVE in the appData folder (Priority 4)

So for *local* testing pre commit/push, you should probably just use the Priority 2 one.  Remember for *live* testing once the pull request is complete, to test on some different projects and to delete or move the symbology files and/or business logic files you were doing *local* testing with.

**Comment by joewheaton** [comment]
This is done.
